apps/src/bounce/bounce.js

```diff
@@ -127,7 +135,7 @@ var initWallMap = function () {
 /**
  * PIDs of async tasks currently executing.
  */
-import * as timeoutList from '../lib/util/timeoutList';
+import * as timeoutList from '../lib/util/timeoutList'; // eslint-disable-line import/order
```

Comment:
Because it seems to be very specifically placed. I think the added context of having it placed here is probably better than having it up with the other imports.